flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/utils/JoinUtil.scala

```diff
@@ -95,6 +105,18 @@ object JoinUtil {
     }
   }
 
+  /** Creates a MiniBatch trigger for join depends on the config. */
+  def createMiniBatchTrigger(config: ReadableConfig): CountCoBundleTrigger[RowData, RowData] = {
```

Comment:
We can reuse the `AggregateUtil#createMiniBatchTrigger` method. Moreover, I think we can extract a MinibatchUtil   class, move minibatch related to utility method to this class.

Reply:
Yes, This would make the code more clear. I would finish it.